Multi cell database/mq adoption (#746)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_deploying-backend-services.adoc

GLANCE_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' GlancePassword:' | awk -F ': ' '{ print $2; }')
HEAT_AUTH_ENCRYPTION_KEY=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' HeatAuthEncryptionKey:' | awk -F ': ' '{ print $2; }')
HEAT_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' HeatPassword:' | awk -F ': ' '{ print $2; }')
HEAT_STACK_DOMAIN_ADMIN_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' HeatStackDomainAdminPassword:' | awk -F ': ' '{ print $2; }')
IRONIC_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' IronicPassword:' | awk -F ': ' '{ print $2; }')
MANILA_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' ManilaPassword:' | awk -F ': ' '{ print $2; }')
NEUTRON_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' NeutronPassword:' | awk -F ': ' '{ print $2; }')
NOVA_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' NovaPassword:' | awk -F ': ' '{ print $2; }')
OCTAVIA_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' OctaviaPassword:' | awk -F ': ' '{ print $2; }')
PLACEMENT_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' PlacementPassword:' | awk -F ': ' '{ print $2; }')

> SWIFT_PASSWORD=$(cat ${TRIPLEO_PASSWORDS[default]} | grep ' SwiftPassword:' | awk -F ': ' '{ print $2; }')

Author: correct

+
<1> Select an existing storage class in your {OpenShiftShort} cluster.

This example provides the required infrastructure database and messaging services for 1 Compute cell
named `cell1`. Adjust the names, counts, IP addresses, and numbers, such as `replicas`, `storage`, or `storageRequest`, as needed.

.Verification

* Verify that MariaDB and RabbitMQ are running for all defined cells:
+
----

> $ RENAMED_CELLS="cell1"

Author: @gibizer this provides an example, how we can extend to multi-cell (as it is shown in follow up PR), yet it still deploys s single cell1 from default tripleo cell

+
<1> Select an existing storage class in your {OpenShiftShort} cluster.

> This example provides required infrastructure database and messaging services for a 1 compute cell
> named `cell1`. Adjust the names, counts, IP addresses, and numbers,
> like `replicas`, `storage`, or `storageRequest`, as needed.

Reviewer: ```suggestion
This example provides the required infrastructure database and messaging services for 1 Compute cell
named `cell1`. Adjust the names, counts, IP addresses, and numbers, such as `replicas`, `storage`, or `storageRequest`, as needed.
```

.Verification

> * Verify that MariaDB and RabbitMQ are running, for all defined cells:

Reviewer: ```suggestion
* Verify that MariaDB and RabbitMQ are running for all defined cells:
```

+
----
$ RENAMED_CELLS="cell1"
$ oc get pod openstack-galera-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
$ oc get pod rabbitmq-server-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
$ for CELL in $(echo $RENAMED_CELLS); do
    oc get pod openstack-$CELL-galera-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
    oc get pod rabbitmq-$CELL-server-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
done
----

> Futher on, we will be referring to the given cells names via an environment
> variable `RENAMED_CELLS`.

Reviewer: ```suggestion
Later on in this guide, the given cells names are referred to by using the environment variable `RENAMED_CELLS`.
```

+
<1> Select an existing storage class in your {OpenShiftShort} cluster.

> This example provides required infrastructure database and messaging services for a 3 compute cells

Author: @jistr only this part is related to other services adoption, indeed
